src/javascript/binary/websocket_pages/user/telegram_bot.js

```diff
@@ -0,0 +1,26 @@
+const TelegramBot = (() => {
+    'use strict';
+
+    const onLoad = () => {
+        const bot_name = 'binary_test_bot';
+        const submit_button = $('#go_to_telegram');
+        submit_button.click((e) => {
```

Comment:
is it better to have an onclick event on the button or onsubmit event on the form? then enter will also trigger submit event, which seems more accessible